src/runtime/gecko/IntlMessageContext.jsm

```diff
@@ -0,0 +1,14 @@
+import { MessageContext } from '../../intl/context';
+
+this.EXPORTED_SYMBOLS = ['MessageContext'];
+
+Components.utils.import('resource://gre/modules/IntlListFormat.jsm');
+Components.utils.import('resource://gre/modules/IntlPluralRules.jsm');
+Components.utils.import('resource://gre/modules/IntlRelativeTimeFormat.jsm');
+
+Intl.PluralRules = PluralRules;
+Intl.ListFormat = ListFormat;
+Intl.RelativeTimeFormat = RelativeTimeFormat;
```

Comment:
I remember you wanted to use lazy getters for these.  I'm okay with not using them for now; just interested to hear what troubles you had with them.